<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>615</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Carlito</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>График</string>
  </property>
  <widget class="QComboBox" name="ChangeColorBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeColorBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>471</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить цвет</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>721</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="graph_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>351</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
